source/docs/software/state-space/state-space-observers.rst

```diff
@@ -0,0 +1,81 @@
+        \mathbf{R})^{-1} \\
+    \hat{\mathbf{x}}_{k+1}^+ &=
+        \hat{\mathbf{x}}_{k+1}^- + \mathbf{K}_{k+1}(\mathbf{y}_{k+1} -
+        \mathbf{C} \hat{\mathbf{x}}_{k+1}^- - \mathbf{D}\mathbf{u}_{k+1}) \\
+    \mathbf{P}_{k+1}^+ &= (\mathbf{I} - \mathbf{K}_{k+1}\mathbf{C})\mathbf{P}_{k+1}^-
+
+.. math::
+    \begin{tabular}{llll}
+      $\mathbf{A}$ & system matrix & $\hat{\mathbf{x}}$ & state estimate vector \\
+      $\mathbf{B}$ & input matrix       & $\mathbf{u}$ & input vector \\
+      $\mathbf{C}$ & output matrix      & $\mathbf{y}$ & output vector \\
+      $\mathbf{D}$ & feedthrough matrix & $\mathbf{\Gamma}$ & process noise intensity
+        vector \\
+      $\mathbf{P}$ & error covariance matrix & $\mathbf{Q}$ & process noise covariance
+        matrix \\
+      $\mathbf{K}$ & Kalman gain matrix & $\mathbf{R}$ & measurement noise covariance
+        matrix
+    \end{tabular}
+
+Covariance (short for correlated variance) is a measurement of how two variables are correlated. In s system with a single state, the covariance matrix is simply :math:`\mathbf{\text{cov}(x_1, x_1)}`, or a matrix containing the variance :math:`\mathbf{\text{var} x_1} of the state :math:`x_1`. The magnitude of this variance is the square of the standard deviation of the Gaussian function describing the current state estimate. Relatively large values for covariance might indicate noisy data, while small covariances might indicate that the filter is more confident about it's estimate. Remember that "large" and "small" values for variance or covariance are relative to the base unit being used -- for example, if :math:`\mathbf{x_1}` was measured in meters, :math:`\mathbf{\text{cov}(x_1, x_1)}` would be in meters squared.
+
+Covariance matrices are written in the following form:
+
+.. math::
+  \mathbf{\Sigma} &= \begin{bmatrix}
+    \text{cov}(x_1, x_1) & \text{cov}(x_1, x_2) & \ldots & \text{cov}(x_1, x_n) \\
+    \text{cov}(x_2, x_1) & \text{cov}(x_2, x_2) & \ldots & \text{cov}(x_1, x_n) \\
+    \vdots         & \vdots         & \ddots & \vdots \\
+    \text{cov}(x_n, x_1) & \text{cov}(x_n, x_2) & \ldots & \text{cov}(x_n, x_n) \\
+  \end{bmatrix}
+
+The state estimate :math:`\mathbf{x}`, together with :math:`\mathbf{P}`, describe the mean and covariance of the Gaussian function that describes our estimate of the system's true state.
+
+The error covariance matrix :math:`\mathbf{P}` describes the covariance of the state estimate :math:`\mathbf{\hat{x}}`. If :math:`\mathbf{P}` is large our uncertainty about the true state is large. Conversely, a :math:`\mathbf{P}` with smaller elements would imply less uncertainty about our true state. In the prediction step, :math:`\mathbf{P}` grows at a rate proportional to the process noise covariance :math:`\mathbf{Q}` and process noise intensity vector :math:`\mathbf{\Gamma}` to show how our certainty about the system's state decreases as we project the model forward. 
+
+Predict step
```

Comment:
Predicting

Reply:
Where? (this is where github's review system falls apart with whole paragraphs per line)

Reply:
A kalman filter has Predict and Correct steps, so it makes sense to title them as such imo
https://en.wikipedia.org/wiki/Kalman_filter#Predict

Reply:
Can't see the whole context for this, but casing the `Correct` with ` might be clearer that its talking about the step named Correct, and not a typo